apps/src/common.rs

```diff
@@ -29,6 +29,8 @@
 //! This module provides some utility functions that are common to quiche
 //! applications.
 
+use std::cmp;
```

Comment:
no need to use this, there's only 2 instances in the whole file so using the full def is fine.